### PR TITLE
(Feature) Claim should be triggered automatically when user is first whitelisted

### PR DESCRIPTION
**Pull request by johnsmith-gooddollar**
# Description

tested with new claim button, fixed issues

About #241 

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

**Comment by roma-hladilka** [comment]
@johnsmith-gooddollar 
could be merged after merging this PR
https://github.com/GoodDollar/GoodWeb3-Mono/pull/22

File: src/pages/gd/Claim/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React, { memo } from 'react'
+import { t } from '@lingui/macro'
+import { useLingui } from '@lingui/react'
+import { ClaimButton, Layout, Title } from '@gooddollar/good-design'
+import { StyleSheet } from 'react-native'
+import { View, Text } from 'native-base'
+import useActiveWeb3React from 'hooks/useActiveWeb3React'
+
+const styles = StyleSheet.create({
```
Comment (johnsmith-gooddollar) [inline src/pages/gd/Claim/index.tsx:9]: protocol ui app uses different approach to style. please look at another components

Comment (johnsmith-gooddollar) [reply src/pages/gd/Claim/index.tsx:9]: @roma-hladilka ^^^

Comment (roma-hladilka) [reply src/pages/gd/Claim/index.tsx]: done

**Review by johnsmith-gooddollar — COMMENTED** [review]
approved

**Pull request merged**